doc/architecture/domains/authorization/features/accept-terms.md

```diff
@@ -0,0 +1,281 @@
+# Terms Acceptance Documentation
+
+## Overview
+
+The Terms Acceptance API allows organizations to accept the latest terms and conditions.
+When an organization accepts the terms, the Authorization database is updated, and an integration event is published,
+to RabbitMQ, using MassTransit's transactional outbox pattern.
```

Comment:
fjern komma så det bliver 'is published to rabbitMq'

Reply:
Done: https://github.com/Energinet-DataHub/energy-origin/commit/65f35263d8e69117f13905eb7d5a03c5d9bf2698

Reply:
Done: https://github.com/Energinet-DataHub/energy-origin/pull/1279/commits/b2fbe037256c6e89c711cbe591be89996f85002f